packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -202,6 +204,10 @@ class IconButton extends StatelessWidget {
   ///    relative to text direction.
   final AlignmentGeometry alignment;
 
+  /// Defines the minimum splash radius on interaction.
+  /// This property must not be null. It defaults to [Material.defaultSplashRadius].
```

Comment:
The general pattern for doc comments is to put a line break between the first sentence and the rest.

Reply:
Done.